syntax = "proto3";

import "corpc_option.proto";

option cc_generic_services = true;

message FooRequest {
    string text = 1;
    int32 times = 2;
}

message FooResponse {
    string text = 1;
    bool result = 2;
}

service FooService {
    option (corpc.global_service_id) = 1;

    rpc Foo(FooRequest) returns(FooResponse);
}
